packages/istanbul-api/lib/reporter.js

```diff
@@ -50,7 +50,13 @@ Reporter.prototype = {
             if (this.config.verbose) {
                 console.error('Create report', fmt, ' with', rptConfig);
             }
-            this.reports[fmt] = libReports.create(fmt, rptConfig);
+            let report;
```

Comment:
I need to make the same changes to nyc. The question is if you want something bigger done.